src/components/AddSenior.tsx

```diff
@@ -84,6 +85,8 @@ const StudentSelector = ({
 };
 
 type SeniorData = {
```

Comment:
Can we potentially reuse the schema instead of redefining the type? What do we think about

```ts
type SeniorData = Omit<z.infer<typeof SeniorPostSchema>, "StudentIDs">
```